### ISSUE TITLE
Conditions and formatting scripts

### ISSUE THREAD

**Issue by arnuschky** [42wim/matterbridge#558]
**Is your feature request related to a problem? Please describe.**
Matterbridge seems to bridge things just fine, for example from Slack to Mattermost. However, we can't filter things and most of the formatting is lost. For example, if I have an app that exclusively integrates with slack, it would be great if I could forward the relevant messages with a specific formatting. Alternatively, one could filter certain messages, for example to bridge all messages that contain a specific keyword.

**Describe the solution you'd like**
Basically something like Zapier for people that know how to write scripts / use the console. So basically two functions:
 - filter stage: decide under which conditions the message is forwarded
 - format stage: decide how / what parts of the message posted on the output protocol

**Describe alternatives you've considered**
Zapier

**Comment by 42wim** [comment]
Check out #731 this should solve your usecase.

**Comment by arnuschky** [comment]
Aweseome, thanks!
